### ISSUE TITLE
Further enhance Nudity setting in Main Menu.

### ISSUE THREAD

**Issue by DioKiriie** [Interkarma/daggerfall-unity#2653]
So I have been thinking on making some mods and just happened to notice that although I have unchecked player nudity in main menu, i still have nude paper doll.

What I propose is to add a distinction between nude and not nude files.
filename.extension would be used as default unless filename_nude.extension would be used if checkbox  is active. 

And with exactly this logic as if one would later on replace sprites with actual models, or some paintings would be with nudity. Mod author can actually distinct one from other without need to replace the only available slots for files, and ship two versions that player can toggle at any moment in menu. And from developer side this would mean that you don't have to write path for nude variant manually like img0202_nude.exntesion or etc.

old mods would still be supported as they just change original files so that is basically it.

**Comment by KABoissonneault** [comment]
I assume that in this case, you change the setting often enough that just switching between different mods each time is bothersome.

Sounds like a plan. I would also rename the setting to cover nudity in general, as the setting also currently covers "Adult Only" quests.

**Comment by DioKiriie** [comment]
Yes I actually make mods and would like to see how they work with other mods, so having two instances of game is not reasonable. I am making both nude and not nude variants and to test them I need to place one then replace with other and so on, while I would like to simply place both files simultaneously and only replace them if I make changes.
And yes something like "Show adult content" would be a great name for an option.
